en/_posts/2017-10-25-rails-girls-berlin-is-growing-up.markdown

```diff
@@ -5,7 +5,7 @@ categories: en blog
 i18n_key: RGBgrowingup
 ---
 
-![](/assets/images/IMG_0876b.jpg)
+![Photo of Ferdous outside, she is smiling and wearing red lipstick](/assets/images/IMG_0876b.jpg)
```

Comment:
ummm, I don't think this is a photo of me. 
it should be something like `Photo of ~50 attendees, coaches and organizers smiling outside. This was taken during the lunch break of one of the Rails beginner's workshops at HTW.`